clifford/test/test_multivector_inverse.py

```diff
@@ -0,0 +1,23 @@
+
+import clifford as cf
+import numpy as np
+import pytest
+
+
+class TestClosedForm:
+
+    @pytest.mark.parametrize('total_dims', [1, 2, 3, 4, 5])
+    def test_hitzer_inverse(self, total_dims):
+
+        def assert_inverse(p, q, Ntests):
+            layout, blades = cf.Cl(p, q)
+            for i in range(Ntests):
+                mv = layout.randomMV()
+                mv_inv = mv.hitzer_inverse()
+                assert np.all(np.abs(((mv * mv_inv) - 1.).value) < 1.e-11)
+
+        for p in range(total_dims+1):
+            q = total_dims - p
+            print("{},{}".format(p, q))
+            assert_inverse(p, q, 100)
```

Comment:
If you find yourself wanting to print what's being tested, you probably wanted a `parametrize`:
```suggestion
    @pytest.mark.parametrize('p q', [
        (p, total_dims-p)
        for total_dims in [1, 2, 3, 4, 5]
        for p in range(total_dims + 1)
    ])
    def test_hitzer_inverse(self, p, q):
        Ntests = 100
        layout, blades = cf.Cl(p, q)
        for i in range(Ntests):
            mv = layout.randomMV()
            mv_inv = mv.hitzer_inverse()
            assert np.all(np.abs(((mv * mv_inv) - 1.).value) < 1.e-11)
```

Reply:
Nice, I wanted to do it with parameterize but didn't think of the double list comprehension